packages/coordinator/src/services/translation/response/release/release-medication-request.ts

```diff
@@ -128,15 +128,15 @@ function createBasedOn(
     url: "https://fhir.nhs.uk/StructureDefinition/Extension-EPS-RepeatInformation",
     extension: [{
       url: "numberOfRepeatsAllowed",
-      valueInteger: new LosslessNumber(parseInt(prescriptionRepeatNumber.high._attributes.value))
+      valueInteger: new LosslessNumber(parseInt(prescriptionRepeatNumber.high._attributes.value).toString())
```

Comment:
This seems wrong, surely if we want a lossless number we should use the original value rather than parsing and immediately stringifying.